src/xhr.js

```diff
@@ -73,7 +73,14 @@ class Xhr {
    * @private
    * @return {!Promise<!FetchResponse>}
    */
-  fetch_() {
+  fetch_(init) {
+    init = init || {};
```

Comment:
This'll cause a weird deopt. You can avoid it by doing `init && init.reponseType ...` below.